docs/rules/no-unknown-wire-adapters.md

```diff
@@ -0,0 +1,39 @@
+## Disallow usage of unknown wire adapters (no-unknown-wire-adapters)
+
+This rule offers a way to restrict the usage of unexpected wire adapters. It can be used to ensure that all the LWC components wired properties are statically analyzable (for example for prefetching or offline purposes).
```

Comment:
```suggestion
This rule offers a way to restrict the usage of unknown wire adapters. It can be used to ensure that all wired properties are statically analyzable in order to support use cases such as prefetching and offline.
```